README.md

```diff
@@ -2,3 +2,10 @@
 
 This repository contains a collection of XML and JSON schemas, used for validators and IDEs that support auto-completion.
 It is the source repository for those schemas. The actual distribution is done via: https://schemas.exasol.com
+
+## Exasol Document Mapping Language (EDML)
+
+The EDML is a JSON-based domain specific language.
+You can use it to configure the mapping of document data to Exasol tables when using a [document virtual schema](https://github.com/exasol/virtual-schema-common-document).
```

Comment:
I just realized that the name of that repo should have been `virtual-schema-common-document-java`.
Too late I am afraid.